core-services/prow/02_config/generate-boskos.py

```diff
@@ -84,6 +84,9 @@
     'metal-perscale-cpt-quota-slice': {
```

Comment:
perf*

Reply:
Its perscale, there is a typo unfortunately in ci-tools